src/main.js

```diff
@@ -5,4 +5,27 @@ import '@fortawesome/fontawesome-free/css/all.min.css'
 import '@fortawesome/fontawesome-free/js/all.min.js'
 import WOW from "wow.js/dist/wow.js";
 
-new WOW().init();
\ No newline at end of file
+new WOW().init();
+
+import Swiper from 'swiper/bundle';
+
+import 'swiper/css/bundle';
```

Comment:
Could we move new imports higher to all other imports?

Reply:
Yes, sir